docs/remove-metrics-server.md

```diff
@@ -0,0 +1,13 @@
+# ⚠️ Remove `metrics-server` Before Using `kube-prometheus`
+
+Before setting up `kube-prometheus`, ensure that the `metrics-server` is **removed** from your Kubernetes cluster to avoid conflicts.
```

Comment:
```suggestion
Before installing `kube-prometheus`, in order to prevent conflicts you should ensure that the `metrics-server` is **removed** from your Kubernetes cluster.
```